namespaces/cloud-platform-live-0.k8s.integration.dsd.io/categorisation-tool-production/resources/s3.tf

```diff
@@ -1,5 +1,5 @@
 module "risk_profiler_digcat_s3_bucket" {
-  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=2.0"
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=bucket_policy"
```

Comment:
didn't know if this should now be changed to ref=2.0 (depends if merged to master?)

Reply:
Yes please, can we please hold off on this one until the PR is merged?

Reply:
This should be version [3.0](https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/releases/tag/3.0) now